### PR TITLE
Change code language to english

### PR DESCRIPTION
**Pull request by Seagullible**
Changing code language from German to English

### CODE REVIEW

**Comment by Seagullible** [comment]
I expect some more work will need to be done on this. I couldn't get the files to actually compile so I could test everything (yet), but since I was just changing variable names I don't foresee any problems

File: src/blueblock/Locator.java [hunk 1]
```diff
@@ -50,51 +50,51 @@ public static Human GetHuman(int x, int y) {
 		return null;
 	}
 
-	public static void MenschGeht(Human h, int Spieler, int Richtung) {
+	public static void GetPlayerPosition(Human h, int Player, int Direction) {
```
Comment (abcdefg30) [inline src/blueblock/Locator.java:53]: This method makes the player move, so I'd rather like `MovePlayer` as name.

**Review by abcdefg30 — COMMENTED** [review]
Looks quite good already. Just need to resolve these small nits and wait for @abc013 to approve.

File: src/blueblock/PowerUp.java [hunk 2]
```diff
@@ -67,7 +67,7 @@ public String GetType() {
 		return Type;
 	}
 
-	public boolean IstAktiv() {
-		return Aktiv;
+	public boolean IstActive() {
```
Comment (abcdefg30) [inline src/blueblock/PowerUp.java:70]: `IsActive`.

File: src/blueblock/Main.java [hunk 3]
```diff
@@ -110,89 +110,89 @@ public Main(int Breite, int H
...
-		fenster.repaint();
-		fenster.setAlwaysOnTop(true);
+		Locator.SetVariables(labels, Player);
+		InfoWindowObject.Refresh();
+		window.repaint();
+		window.setAlwaysOnTop(true);
 	}
 
-	public static void FarbeWechseln(Label label, Color Color) {
+	public static void colorChange(Label label, Color Color) {
```
Comment (abcdefg30) [inline src/blueblock/Main.java:174]: I think `ChangeColor` sounds better.

**Comment by abcdefg30** [comment]
Hm, also looks like this needs a rebase. Are you familiar with rebasing?

File: src/blueblock/Main.java [hunk 4]
```diff
@@ -110,89 +110,89 @@ public Main(int Breite, int H
...
 		 * New Idea for future: label.setText("OOO");
 		 * label.setForeground(Color);
 		 */
 		label.setBackground(Color);
 	}
 
-	public void Offen(boolean offen) {
-		fenster.setVisible(offen);
-		InfoFenster.offen(offen);
+	public void onStart(boolean onStart) {
```
Comment (abc013) [inline src/blueblock/Main.java:182]: I think this should be named `setOpen` or something like this.

Comment (abc013) [inline src/blueblock/Main.java]: I think this should be named `setOpen` or something like this.

File: src/blueblock/InfoWindow.java [hunk 5]
```diff
@@ -113,18 +113,18 @@ public void actionPerformed(ActionEvent arg0) {
...
 		GreenBlockposition = "Platz: - ";
 		CyanBlockposition = "Platz: - ";
 		MagentaBlockposition = "Platz: - ";
-		InfoFenster.repaint();
+		InfoWindowObject.repaint();
 	}
 
-	public void offen(boolean offen) {
-		InfoFenster.setVisible(offen);
+	public void onStart(boolean onStart) {
```
Comment (abc013) [inline src/blueblock/InfoWindow.java:126]: Same as above.

**Review by abc013 — COMMENTED** [review]
First, thanks for the Pull request etc.!
After the fixes from you we can merge this! 👍

File: src/blueblock/Human.java [hunk 6]
```diff
@@ -11,29 +11,29 @@
 			new Color(155, 0, 155) };
 	private Label[][] labels = Main.labels;
 	private int pos1, pos2, Dpos1, Dpos2;
-	public int Kills, schritte, SuperScore, PowerUps, Spieler;
-	private boolean lives, poisoned, gesch�tzt;
+	public int Kills, steps, SuperScore, PowerUps, Player;
+	private boolean lives, poisoned, guarded;
```
Comment (abcdefg30) [inline src/blueblock/Human.java:15]: I just noticed `protected` would fit more ("guared" is more like "bewacht" than "beschützt", I think).

**Comment by Seagullible** [comment]
Git isn't playing nice with my branches so I'm just going to make a new pull request -_-'

File: src/blueblock/Human.java (hunk 6 above)
Comment (Seagullible) [reply src/blueblock/Human.java:15]: I actually thought this at first as well, but `protected` is a keyword in java :(

**Pull request closed without merging**